charts/lighthouse/templates/webhooks-ingress.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.webhooks.ingress.enabled }}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ template "webhooks.name" . }}
```

Comment:
For legacy and Prow compatibility reasons, the ingress name still needs to be `hook`. That's why https://github.com/jenkins-x/lighthouse/blob/master/test/e2e/helpers.go#L347 failed in the e2e tests.